system_strings/sysmes_text.en

```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
-気分転換になるだろう。^指定された廃工場へと足を運び、^促されるまま扉を開けた。
+// First part from 22850.
+// @flowchart_descr
+This could be a nice change of pace.
+ I make my way to the abandoned factory, and open the door as instructed.
 }
 [sha:33b5d8137384f5e8f133e264644f945488093b9b]{
 -- 因果応報、逃れる道はない。^吸血鬼を殺すと決めた以上、^この手は血に濡れていく。
-因果応報、逃れる道はない。^吸血鬼を殺すと決めた以上、^この手は血に濡れていく。
+// 因果応報 matched to 42720.
+// @flowchart_descr
+I will give him his just desserts.
+ There is no going back now.
+ The moment I resolved to kill a vampire, my hands were stained with blood.
 }
 [sha:6d3521330894be81fbce4e6c45e1ea3400b04eda]{
 -- 悪趣味には付き合いきれない。^だが、その拒絶は彼女にとって^納得のいくものではなかった。
-悪趣味には付き合いきれない。^だが、その拒絶は彼女にとって^納得のいくものではなかった。
+// Cribbed from 23021.
+// @flowchart_descr
+This is messed up. I want to part in it.
+ But Noel isn't satisfied at all by my refusal.
 }
 [sha:74a545322db885f4f945b43d9309406b93045bb5]{
 -- ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
-ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
+// @flowchart_descr
+Noel and I exit the building into a back alley.
+ She starts to tell me about Ciel's past, delighting in my reactions.
 }
 [sha:b33017f4101f45e5763f01a2eefd88258cd57822]{
 -- 沈んだ気持ちで部屋に戻る。^先輩の忠告と、あいつとの約束。^二つの間で心が揺れている――
-沈んだ気持ちで部屋に戻る。^先輩の忠告と、あいつとの約束。^二つの間で心が揺れている――
+// @flowchart_descr
+I return to my room in low spirits.
+ Senpai issued me a warning, but I made a promise with Arcueid.
+ I can't make up my mind between the two...
 }
 [sha:853098cc87b3732c8375f362476529c8bd00d367]{
 -- 約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
-約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
+// @flowchart_descr
+The promised time has arrived.
+ I'll meet up with Arcueid one more time, and confirm how I really feel.
 }
 [sha:0254e07a578acf4d16cf7b673be4c3c9db860126]{
 -- このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
-このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
+// @flowchart_descr
+I can't let things carry on like this.
+ I need to be blunt and ask Ciel-senpai to go back home.
 }
 [sha:117c5ea4b2c0cc46d7a917c15924d0798453ccbd]{
 -- 先輩は好きにしろと言った。^ここは冷静に言い返して、^シエル先輩にお帰り願おう。
-先輩は好きにしろと言った。^ここは冷静に言い返して、^シエル先輩にお帰り願おう。
+// @flowchart_descr
+Senpai told me I could do as I wish.
+ Now's the time to be logical, and ask Ciel-senpai to go back home.
 }
 [sha:9bf5309044ffd5661bf78f1c55e91c7dd36eacd1]{
 -- 公園に到着してしまった。^竜虎相打つ。^最悪の事態は容赦なく訪れる。
-公園に到着してしまった。^竜虎相打つ。^最悪の事態は容赦なく訪れる。
+// @flowchart_descr
+For better or worse, I've arrived at the park.
+ Now comes the clash of the titans.
+ Things are rapidly going south.
 }
 [sha:b69f7eb5d3de09b8f6774926152c6fb3d4c8e2a7]{
 -- 激突する代行者と吸血鬼。^トップランカーの戦いが、^ここに。
-激突する代行者と吸血鬼。^トップランカーの戦いが、^ここに。
+// @flowchart_descr
+The clash between Executor and vampire commences.
+ I'm witnessing a battle for the ages.
 }
 [sha:d7a92a84ed79beb1269075e32635b92eb0bcad84]{
 -- 伯仲している二人の戦闘能力。^その均衡は徐々に崩れてゆき、^そして、ついに――
-伯仲している二人の戦闘能力。^その均衡は徐々に崩れてゆき、^そして、ついに――
+// @flowchart_descr
+Their abilities start out evenly matched.
+ But that balance slowly crumbles, until suddenly――
```

Comment:
'start out' seems redundant given the next sentence.
is it really sudden if it happens slowly?
>Their abilities are evenly matched.
>But that balance slowly crumbles, until at last——

Reply:
Hmm, I feel like it does need that 'start out' lead into establish the existence of something to then tip.

Reply:
I don't really see the need